news/2025/2025-02-13-osumania-7k-world-cup-2025-semifinals-recap.md

```diff
@@ -0,0 +1,285 @@
+
+And what better way to start the match than by picking [Elliot Hsu - A Guiding Star](https://osu.ppy.sh/beatmapsets/2317897#mania/4964065) (LN1), a category proven to be most divisive throughout the tournament. Still, it was a surprisingly close point for both of our teams whose long note skills are on point, a fact completely unrelated to the existence of the category in this year's pool — *no relation whatsoever~*. Snarky tone aside, some of the scores were worth highlighting just for how impressive they were. Take for example ::{ flag=CN }:: [Watch01](https://osu.ppy.sh/users/13967954)'s 985,571 full combo, which was the second best score on that map during the weekend!
+
+Team BIG MAXUS was no pushover, however. With ::{ flag=AU }:: [oct4](https://osu.ppy.sh/users/10303887) bathing in the limelight with their performance on rice maps such as [Lime - Pixel Planet](https://osu.ppy.sh/beatmapsets/2317895#mania/4964061) (RC6), or with ::{ flag=CN }:: [_Yiiiii](https://osu.ppy.sh/users/6066359) handling themselves quite well in [Trina Lydia - TETRATONE ALTiSSiMO](https://osu.ppy.sh/beatmapsets/2317891#mania/4964056) (LN3) with a team-leading score of 944,114 points. Shoutouts to ::{ flag=KR }:: [rubyy](https://osu.ppy.sh/users/9880422) who was consistently top scoring in many of the maps they played in! Makes you wonder how many of these players would've bolstered their team's rosters on this year's MWC.
+
+But this wouldn't be a showcase without showing the tiebreaker, wouldn't it? So of course [kanemiko - Ruin in Cascade](https://osu.ppy.sh/beatmapsets/2318032#mania/4964418) had to make an appearance at the very end of the showcase to determine who would be the biggest of ETs after the match! A match so engaging that even the commentators couldn't help themselves by cursing one of the players in the middle of the map:
+
+<div align="center" class="osu-md__paragraph">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/media/news/2025-02-14-watch-this.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+Given the match's length, we haven't been able to recap it in full. Because of that, we strongly suggest you to [watch the whole match](https://www.twitch.tv/videos/2375746661). Having fun in a competitive environment is part of what makes a game enjoyable. If it's not fun, why bother? So go grab a handful of buddies of yours, download any of the pools, and start having a blast together on a multiplayer lobby!
```

Comment:
```suggestion
Given the match's length, we haven't been able to recap it in full. Because of that, we strongly suggest you to [watch the whole match](https://www.twitch.tv/videos/2375746661). Having fun in a competitive environment is part of what makes a game enjoyable. If it's not fun, why bother? Go grab a handful of buddies of yours, download any of the pools, and start having a blast together in a multiplayer lobby!
```